swagger_server/jobs/pull_topo_changes.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import json
+import logging
+import os.path
+import sys
+import threading
+import time
+import urllib.request
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
+)
+
+from messaging.rpc_queue_producer import *
+from utils.db_utils import *
+
+DOMAIN_CONTROLLER_PULL_URL = os.environ.get("DOMAIN_CONTROLLER_PULL_URL")
+DOMAIN_CONTROLLER_PULL_INTERVAL = os.environ.get("DOMAIN_CONTROLLER_PULL_INTERVAL")
+logger = logging.getLogger(__name__)
+
+
+def main():
+    db_instance = DbUtils()
+    db_instance.initialize_db()
+
+    process_domain_controller_topo(db_instance)
+
+
+def process_domain_controller_topo(db_instance):
+    while True:
+        latest_topology_exists = False
+        latest_topology = db_instance.read_from_db("latest_topology")
+
+        if latest_topology:
+            latest_topology_exists = True
+            json_latest_topology = json.loads(latest_topology["latest_topology"])
```

Comment:
Wonder what we would get if we do `json.loads(latest_topology)` -- what is the shape of the document stored in the DB?

Reply:
The latest_topology is a json object, it has a json string stored as the value to ["latest_topology"]. Each time when lc updates the topology, it'll save the latest topology inside here.

Reply:
I wonder if some error checking would be useful?  Seems that we are assuming to be some data coming from an external source always to be correct.  That `json.loads()` can raise an error when it can't handle the input.